packages/button/src/ButtonStyles.tsx

```diff
@@ -31,14 +31,14 @@ const sizes = {
   `,
   sm: css<InjectedProps>`
     line-height: 1em;
-    font-size: ${({ theme }) => theme.fontSizesMap.xxs}px;
+    font-size: ${({ theme }) => theme.fontSizesMap.xs}px;
     border-radius: ${({ theme }) => theme.borderRadiusesMap.sm}px;
     padding: ${({ $square }) => ($square ? '16px' : '16px 24px')};
     min-width: ${({ $square }) => ($square ? '0' : '100px')};
   `,
   md: css<InjectedProps>`
     line-height: 1em;
-    font-size: ${({ theme }) => theme.fontSizesMap.xs}px;
+    font-size: ${({ theme }) => theme.fontSizesMap.sm}px;
```

Comment:
Discussed with Stanislav, found out that sizes for these buttons are incorrect. [Proofs](https://www.figma.com/file/0j4EJ3Ra7OPBeEKLXFQiGM/LIDO-Design-System?node-id=10497%3A2582)